Packs/GCP/Integrations/GCP/GCP.py

```diff
@@ -379,6 +471,473 @@ def compute_firewall_patch(creds: Credentials, args: dict[str, Any]) -> CommandR
     return CommandResults(readable_output=hr, outputs_prefix="GCP.Compute.Operations", outputs=response)
 
 
+def storage_bucket_list(creds: Credentials, args: dict[str, Any]) -> CommandResults:
+    """
+    Retrieves the list of buckets in the project associated with the client.
+
+    Args:
+        creds (Credentials): GCP credentials.
+        args (dict[str, Any]): Command arguments including optional project_id, max_results, prefix, page_token.
+
+    Returns:
+        CommandResults: List of buckets with their metadata.
+    """
+    project_id = args.get("project_id")
+    max_results = arg_to_number(args.get("max_results"))
+    prefix = args.get("prefix")
+    page_token = args.get("page_token")
+    demisto.debug(f"[GCP: storage_bucket_list] \nMax results: {max_results}, \nPrefix: {prefix}, \nPage token: {page_token}")
+
+    storage = GCPServices.STORAGE.build(creds)
+
+    # Build request parameters
+    request_params = {"project": project_id, "maxResults": max_results, "prefix": prefix, "pageToken": page_token}
+
+    remove_nulls_from_dictionary(request_params)
+
+    demisto.debug(f"[GCP: storage_bucket_list] Request params: {request_params}")
+    response = storage.buckets().list(**request_params).execute()  # pylint: disable=E1101
+
+    buckets = response.get("items", [])
+    demisto.debug(f"[GCP: storage_bucket_list] Buckets returned: {len(buckets)}")
+    bucket_data: list[dict[str, Any]] = []
+    hr_bucket_data: list[dict[str, Any]] = []
+
+    for bucket in buckets:
+        bucket_info = {
+            "Name": bucket.get("name"),
+            "TimeCreated": bucket.get("timeCreated"),
+            "TimeUpdated": bucket.get("updated"),
+            "OwnerID": bucket.get("owner", {}).get("entityId", ""),
+            "Location": bucket.get("location"),
+            "StorageClass": bucket.get("storageClass"),
+        }
+        bucket_data.append(bucket_info)
+
+        hr_bucket_data.append(
+            {
+                "Name": bucket_info["Name"],
+                "TimeCreated": _format_gcp_datetime(bucket_info["TimeCreated"]),
+                "TimeUpdated": _format_gcp_datetime(bucket_info["TimeUpdated"]),
+                "OwnerID": bucket_info["OwnerID"],
+                "Location": bucket_info["Location"],
+                "StorageClass": bucket_info["StorageClass"],
+            }
+        )
+
+    hr = tableToMarkdown("GCP Storage Buckets", hr_bucket_data, removeNull=True)
```

Comment:
```suggestion
    hr = tableToMarkdown("GCP Storage Buckets", hr_bucket_data, removeNull=True, headerTransform=pascalToSpace)
```
Add the argument `headerTransform=pascalToSpace` for all hr.